src/Illuminate/Http/Middleware/SetCacheHeaders.php

```diff
@@ -31,6 +31,18 @@ public function handle($request, Closure $next, $options = [])
         if (isset($options['etag']) && $options['etag'] === true) {
             $options['etag'] = md5($response->getContent());
         }
+        
+        if (isset($options['last_modified']))
+		{
```

Comment:
You should use spaces to indent, here it looks like there are some tabs involved